wallet/src/Pos/Wallet/Web/Api.hs

```diff
@@ -298,6 +300,19 @@ type UpdateTx =
     :> ReqBody '[JSON] CTxMeta
     :> WRes Post ()
 
+type CancelApplyingPtxs =
+       "txs"
+    :> "resubmission"
+    :> "cancel"
+    :> WRes Get ()
```

Comment:
This was not very clear from the spreadsheet, but these two endpoints are not RESTful-abiding and really they should be `POST`, not `Get`, as they are not idempotent 😉

Reply:
Ah, I see. Let me do that, and resolve the merge conflicts!